### PR TITLE
Fixing the card from #024: fixing managed team page

### PR DESCRIPTION
**Pull request by devfabien**
# PR Description

This PR includes the fixing of design with the purpose of insuring the quality assurance of the website

## Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style

- [x] #024 : fixing managed team page
- [x] #026 : fixing the Qualitat section
- [x] #027 : fixing founder page

## Description of tasks that were expected to be completed and that are not yet completed

- [x] None

## How this may be tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. run `npm i` to install all required dependencies
2. run `npm run build`
3. run `npm run start`
4. step 4...

## Screenshots (If appropriate/applicable)

Provide screenshots related to your PR in case it's appropriate and applicable
Screenshot Capture - 2023-08-24 - 16-18-31

Screenshot Capture - 2023-08-24 - 16-19-24

Screenshot Capture - 2023-08-24 - 16-20-37

Screenshot Capture - 2023-08-24 - 16-21-21

## Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

File: src/sections/managedTeam.html [hunk 1]
```diff
@@ -1,14 +1,15 @@
 <body class="bg-backgroundColor">
-    <div class="relative bg-white py-20 md:px-16 px-6" style="-webkit-clip-path: polygon(0 0, 100% 0, 100% 100%, 0 95%);
+    <div class="relative bg-secondary
+     py-20 md:px-28 px-6" style="-webkit-clip-path: polygon(0 0, 100% 0, 100% 100%, 0 95%);
     clip-path: polygon(0 0, 100% 0, 100% 100%, 0 95%);">
-        <p class="text-gray-500 font-bold font-title py-2 text-xs">Managed Teams</p>
+        <p class="text-gray-500 font-bold font-title py-2">Managed Teams</p>
```
Comment (kunda4) [inline src/sections/managedTeam.html]: add font size here

File: src/sections/managedTeam.html [hunk 2]
```diff
@@ -32,20 +33,20 @@ <h2 class="font-bold font-title">Ihre <span
             </div>
 
         </div>
-        <p class="text-gray-500 font-semibold font-title py-2 text-sm"> African Team Box</p>
+        <p class="text-gray-500 font-bold font-title py-2"> African Team Box</p>
```
Comment (kunda4) [inline src/sections/managedTeam.html]: Add font size here too

**Review by kunda4 — CHANGES_REQUESTED** [review]
Good job Fabien, u did great but there is some litle things you have to care about it. happy coding!

File: src/sections/managedTeam.html (hunk 1 above)
Comment (devfabien) [reply src/sections/managedTeam.html]: i added the font size

File: src/sections/managedTeam.html (hunk 2 above)
Comment (devfabien) [reply src/sections/managedTeam.html]: I added the font size

**Review by kareraolivier — APPROVED** [review]
Finish agnes' requests

**Review by kunda4 — APPROVED** [review]
Good Job

**Pull request merged**